schemas/users.js

```diff
@@ -0,0 +1,12 @@
+import { object, string } from 'yup';
+
+export const userRegistrationSchema = object({
+  fullname: string().required().min(3),
+  email: string().email().required().nonNullable(),
```

Comment:
NVM It will serve the purpose of Regex?

Reply:
Yes, I am using Yup library for data validation.